music21/instrument.py

```diff
@@ -79,6 +78,8 @@ def unbundleInstruments(streamIn: StreamType, *, inPlace=False) -> Optional[Stre
 
     if inPlace is False:
         return s
+    else:
+        return None
```

Comment:
can we disable this setting?  One of my least liked parts of TypeScript is having to return undefined if I return something else somewhere.  None is Python's default return value.  I'd rather not have a lot of "return None" at the end of files.

Reply:
Will do (and for an entertaining read see https://github.com/python/mypy/issues/7511)

Reply:
If it was anyone but Guido, his "whatever" comment would be deleted for trolling.  I'm with others -- mypy is a type checker not a style enforcer.